requirements.txt

```diff
@@ -2,3 +2,5 @@ jwcrypto
 six
 python-etcd
 requests
+docker-py
+configparser; python_version < '3.4'
```

Comment:
Python 3.4+ have configparser with new interpolation.

Do we support 3.3? If not I'll replace it with < 3